### PR TITLE
Typos etc in chapter 5

### PR DESCRIPTION
**Pull request by PublicHealthDataGeek**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Many thanks @PublicHealthDataGeek, good to go?

**Comment by PublicHealthDataGeek** [comment]
I am not sure if I review what you have done (cant see a review button, do I need to go back in to the chapter page and read it again) or whether I just 'comment' back to you or 'close with comment' ?

**Comment by Robinlovelace** [comment]
> I am not sure if I review what you have done (cant see a review button, do I need to go back in to the chapter page and read it again) or whether I just 'comment' back to you or 'close with comment' ?

I just meant to ask if the changes were ready to merge from your perspective. You can also request a review but I'll assume all PRs are ready to review unless otherwise stated. Many thanks!

File: 05-data.Rmd [hunk 1]
```diff
@@ -40,30 +40,30 @@ crashes = tibble(
...
-crashes %>% filter(casualty_age > 50) # filter rows
+crashes %>% filter(casualty_age > 50) # filters rows
 crashes %>% select(casualty_type) # select just one column
 ```
 
 It should be clear what happened: `filter()` returns only rows that match the criteria in the function call, only observations with a `casualty_age` greater than 50 in this case.
 Likewise, `select()` returns data objects that include only columns named inside the function call, `casualty_type` in this case.
 
-To gain a greater understanding of the functions type and run the following commands:
+To gain a greater understanding of the functions type and run the following commands, which also illustrate how the `%>%` can be used more than once to manipulate data (more on this soon):
```
Comment (Robinlovelace) [inline 05-data.Rmd:66]: Good clarification.

File: 05-data.Rmd [hunk 2]
```diff
@@ -134,12 +132,13 @@ crashes %>% select(ca) # press Tab when your cursor is just after the a
 
 You should see `casualty_age` and `casualty_type` pop up as options that can be selected by pressing Up and Down.
 This may not seem like much, but when analysing large datasets with dozens of variables, it can be a godsend.
-The chapter has not provided a comprehensive introduction to the `tidyverse` fleet of packages but should provide enough to get started with using it for road safety data analysis.
+
+This chapter has not provided a comprehensive introduction to the `tidyverse` suite of packages but should provide enough to get started with using it for road safety data analysis.
 For further information, check out up-to-date online courses from respected organisations like [Data Carpentry](https://datacarpentry.org/R-ecology-lesson/index.html) and the free online [books](https://bookdown.org/) such as [R for Data Science]((https://r4ds.had.co.nz/)) [@grolemund_data_2016].
 
-## Tidyverse exersises
+## Tidyverse exercises
```
Comment (Robinlovelace) [inline 05-data.Rmd:139]: :+1:  cannot believe how many typos there were :facepalm:

**Review by Robinlovelace — COMMENTED** [review]
Great PR Caroline!

**Pull request merged**